zhusuan/distributions/multivariate.py

```diff
@@ -403,3 +372,254 @@ def _log_prob(self, given):
 
     def _prob(self, given):
         return tf.exp(self._log_prob(given))
+
+
+class ExpConcrete(Distribution):
```

Comment:
Maybe Chi2 and Exponential are more suitable to use inheritance if we want to add them, since they are both a special case of Gamma